src/Events/Authorization/SubscriptionXacmlMapper.cs

```diff
@@ -72,7 +72,7 @@ private static XacmlJsonCategory CreateResourceCategory(Subscription subscriptio
 
             if (isAppEventSubs)
             {
-                string[] resourceAttParts = resourceFilterValue.Split('.');
+                string[] resourceAttParts = resourceFilterValue.Split('_');
```

Comment:
Could it be that apps are more likely to have '_' in their name? Is it possible to have '_' in other parts of the urn

Reply:
nvm, tried to create an app with _, not allowed